packages/webpack/src/presets/base.ts

```diff
@@ -88,20 +87,19 @@ function basePlugins (ctx: WebpackConfigContext) {
       server: 'orange',
       modern: 'blue',
     }
-    ctx.config.plugins.push(new WebpackBar({
+    ctx.config.plugins.push(new WebpackBarPlugin({
       name: ctx.name,
       color: colors[ctx.name as keyof typeof colors],
       reporters: ['stats'],
-      // @ts-expect-error TODO: this is a valid option for Webpack.ProgressPlugin and needs to be declared for WebpackBar
       stats: !ctx.isDev,
       reporter: {
         reporter: {
-          change: (_, { shortPath }) => {
+          change: (_: any, { shortPath }: any) => {
```

Comment:
any reason we don't have types for these?

Reply:
I'm not sure, after changing to importing from `#builder`, the typecheck just throw error in those place.

My Vscode's intellisense is also showing the `WebpackBarPlugin` as `any` type.

![image](https://github.com/user-attachments/assets/6c60e7fe-419b-4ff3-9ff4-becfa2eb4b3e)